openspec/changes/project-02-plan-root-command-fix/design.md

```
@@ -0,0 +1,4 @@
+## Design
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)